### PR TITLE
Fixing incorrect resource name for modernizr.js (#777)

### PR DESCRIPTION
**Pull request by tgoetz**
See https://github.com/l0rdn1kk0n/wicket-bootstrap/issues/777

### CODE REVIEW

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/references/ModernizrJavaScriptReference.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ public static ModernizrJavaScriptReference instance() {
      * Private constructor.
      */
     private ModernizrJavaScriptReference() {
-        super("/modernizr/current/modernizr.js");
+        super("/modernizr/current/modernizr.min.js");
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/references/ModernizrJavaScriptReference.java:34]: Is this really needed ? Wicket should use the min version in PROD mode

Comment (tgoetz) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/references/ModernizrJavaScriptReference.java:34]: I have not tested in PROD mode yet, but in DEV mode I get the mentioned exception ...

Comment (tgoetz) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/references/ModernizrJavaScriptReference.java:34]: I'd say the real question is: why does the webjars artifact of modernizr-2.8.3-1 not contain modernizr.js? modernizr-2.8.3 contains both (modernizr.js and modernizr.min.js) ...

Comment (tgoetz) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/references/ModernizrJavaScriptReference.java:34]: @martin-g what do you think? I currently created a workaround for this in my current project (created my own ModernizrJavaScriptReference with the currected path and configured a global resource replacement in Wicket).

**Comment by martin-g** [comment]
I'd rather go back to the earlier version with both files.

**Comment by tgoetz** [comment]
Ok, this is also a suitable solution I guess. I thought that upgrading modernizr to 2.8.3-1 might have been done for a reason ...?! But if you say it's safe to go back to 2.8.3 I will provide a pull request for this.

**Pull request closed without merging**